src/pages/workspace/taxes/WorkspaceTaxesSettingsWorkspaceCurrency.tsx

```diff
@@ -0,0 +1,72 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import type {ListItem} from '@components/SelectionList/types';
+import TaxPicker from '@components/TaxPicker';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {setWorkspaceCurrencyDefault} from '@libs/actions/TaxRate';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceTaxesSettingsWorkspaceCurrencyProps = WithPolicyAndFullscreenLoadingProps &
+    StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.TAXES_SETTINGS_WORKSPACE_CURRENCY_DEFAULT>;
+
+function WorkspaceTaxesSettingsWorkspaceCurrency({
+    route: {
+        params: {policyID},
+    },
+    policy,
+}: WorkspaceTaxesSettingsWorkspaceCurrencyProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const defaultTaxName =
+        (policy?.taxRates?.defaultExternalID &&
+            `${policy?.taxRates.taxes[policy?.taxRates?.defaultExternalID].name} (${policy?.taxRates.taxes[policy?.taxRates?.defaultExternalID].value}) • ${translate('common.default')}`) ??
```

Comment:
Same here, it would be cleaner to extract it to a variable.

Reply:
Done